common/agentinstall/params.go

```diff
@@ -30,7 +27,7 @@ func WithLatest() func(*Params) error {
 	}
 }
 
-// WithVersion use a specific version of the Agent. For example: `6.39.0` or `7.41.0-rc.7`
+// WithVersion use a specific version of the Agent. For example: `6.39.0` or `7.41.0~rc.7-1
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
We still support rc candidate versions right ? Not used to the `~` format, I only used `-` rc candidates

Reply:
A RC candidate version has a format similar to `7.41.0~rc.7-1`. Note: the Agent image docker version uses `-` format.